modules/configuring-kernal-page-size.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-page-size-optimization_{context}"]
+= Configuring kernel page sizes
+
+Use the `kernelPageSize` specification in a performance profile to configure the kernel page size on a specific node. Specify larger kernel page sizes for memory-intensive, high-performance workloads. 
+
+[NOTE]
+====
+For nodes with an x86_64 or AMD64 architecture, you can only specify `4k` for the `kernelPageSize` specification. For nodes with an AArch64 architecture, you can specify `4k` or `64k` for the `kernelPageSize` specification. The default value is `4k`.
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a performance profile to target nodes where you want to configure the kernel page size:
+
+.. Create a YAML file that defines the `PerformanceProfile` resource:
```

Comment:
Feels a little funny to have a single-step procedure (which makes me wonder if technically this should be a `*`?) with two substeps.

Could this just be flattened out to two normal steps? Makes the sentence kinda long, but something like:
```suggestion
. Create a performance profile to target nodes where you want to configure the kernel page size by creating a YAML file that defines the `PerformanceProfile` resource:
```